server/interceptor/interceptor.go

```diff
@@ -111,7 +111,7 @@ func initMetrics(mf monitoring.MetricFactory) {
 	requestCounter = mf.NewCounter(
 		"interceptor_request_count",
 		"Total number of intercepted requests",
-		monitoring.TreeIDLabel)
+		monitoring.TreeIDLabel, "quota_user")
```

Comment:
Changing the labels for an existing metric means that any existing monitoring will need to be changed to aggregate over `quota_user`; is that going to cause problems?